test/test_bug3473.m

```diff
@@ -0,0 +1,95 @@
+function test_bug3453
```

Comment:
test scripts should have WALLTIME and MEM specifications at the top.

Reply:
Add WALLTIME and MEM to the script